source/schema.txt

```diff
@@ -335,6 +335,28 @@ To delete a location filter from the map:
 
 .. |query-build-type| replace:: :ref:`build-query`
 
+Troubleshooting
+---------------
+
+If the analysis of your schema times out, it might be because the 
+collection you are analyzing is very large and the operation is stopped 
```

Comment:
"is stopped" sounds a little strange to me here. I think it could sound better if we reword -- how about something like:

```suggestion
collection you are analyzing is very large, causing MongoDB to stop the operation 
```

Reply:
Agreed, took that suggestion